apps/product/src/main/java/com/ecommerce/product/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.ecommerce.product.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler({RuntimeException.class})
+    public ResponseEntity<Object> handleRuntimeException(RuntimeException exception) {
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(exception.getMessage());
```

Comment:
You should not send all the system exception message to end user. Just a simple message like "Internal server error" is enough.